.github/workflows/build.yaml

```diff
@@ -1,6 +1,10 @@
 name: build
 
 on:
+  # Necessary when pull requests are updated.
+  push: {}
```

Comment:
What does this do? I imagine the idea is to take certain actions when code is pushed, but since we haven't defined any particular actions it's unclear why this change is necessary just yet.